_posts/2020-12-03-How-to-create-a-file.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Creating A File
+description: 
+summary: 
+tags:
+---
+Today is the week after thanksgiving break and Kajari instructed us to work on starting our website. To create a new file, first you have to go to Finder and manually create a file. Then you have to go to terminal and from there you have to type in code so later in sublime text you can see the files inside the folder you have created. Then, you go into sublime text and you can open that folder and you can see where you can start with the coding. WIth this done, you can go into chrome and open file and then open the index.html. 
+
+To create the files, in terminal I opened Projects, then I 'mkdir'change (which is the name of the file), then I opened the file which is called 'change'. Then i wrote out 'touch index.html' then I opened my file again and then typed in 'touch README.md' which followed 'touch script.js' and 'touch style.css'. Next time, be careful what you are typing because your folder may have a different name and not change.
```

Comment:
`mkdir` is to create a folder, not a file.

Reply:
Instead of prose, please make it into a checklist that you can follow. Take a look at Osvaldo's post here: https://osvaldo178.github.io/2020/12/03/how-create-a-new-index-html-file